eth/filters/filter_system.go

```diff
@@ -153,7 +153,7 @@ func NewEventSystem(backend Backend, lightMode bool) *EventSystem {
 	if m.txsSub == nil || m.logsSub == nil || m.rmLogsSub == nil || m.chainSub == nil || m.pendingLogsSub == nil {
 		log.Crit("Subscribe for event system failed")
 	}
-	if m.voteSub == nil {
+	if m.voteSub == nil || m.finalizedHeaderSub == nil {
 		log.Warn("Subscribe for vote event failed")
```

Comment:
The log should be log.Warn("Subscribe for vote or finalized header event failed")

Reply:
Fixed